k8s/perftest/common/probate/orchestrator-service.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-orchestrator-service
+  namespace: probate
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: probate-orchestrator-service
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    name: probate-orchestrator-service
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    version: 1.0.0
+  values:
+    tags:
+      idam-pr: false
```

Comment:
this should be handled in the chart with having this off by default and only enabled on preview.

not a blocker